res/css/views/rooms/_WhoIsTypingTile.scss

```diff
@@ -23,7 +23,7 @@ limitations under the License.
 
 /* position the indicator in the same place horizontally as .mx_EventTile_avatar. */
 .mx_WhoIsTypingTile_avatars {
-    flex: 0 0 83px; // 18 + 65
+    flex: 0 0 82px; // 18 + 64
```

Comment:
likewise here

Reply:
Also this seems to affect more than just the timeline card, is that intentional and tested?

Reply:
Yes, it's intentional, I noticed while working on this that the typing tile in the main timeline was 1px out of alignment.